src/contribute-to-tidb/report-an-issue.md

```diff
@@ -0,0 +1,48 @@
+
+  - Minimal reproduce step.
+  - What did you expect to see?
+  - What did you see instead?
+  - What is your TiDB version?
+
+Answering these questions give the details about your problem so other contributors or TiDB users could pick up your issue more easily. 
+
+As previous section shows, duplicated issues should be reduced. To help others who encountered the problem find your issue, except for problem details answered in the issue template, a descriptive title which contains information that might be unique to it also helps. This can be the components your issue belongs to or database features used in your issue, the conditions that trigger the bug, or part of the error message if there is any. 
+
+## Making good issues
+
+Except for a good title and detailed issue message, you can also add suitable labels to your issue via [\label](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#type), especially which component the issue belongs to. Many committers and contributors only focus on certain subsystems of TiDB. Setting the appropriate component is important for getting their attention.
+
+If you are able to, you should take more considerations on your issue:
+
+  - Does the feature fit well into TiDB's architecture? Will it scale and keep TiDB flexible for the future, or will the feature restrict TiDB in the future?
+  - Is the feature a significant new addition (rather than an improvement to an existing part)? If yes, will the community commit to maintaining this feature?
+  - Does this feature align well with currently ongoing efforts?
+  - Does the feature produce added value for TiDB users or developers? Or does it introduce the risk of regression without adding relevant user or developer benefit? 
+
+Deep thoughts could help the issue proceed faster and help build your own reputation in the community.
+
+## Understanding the issue’s progress and status
+
+Once your issue is created, other contributors might take part in. You can [\cc](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#cc) reviewers to request reviews. You need to discuss with them, provide more information they might want to know, address their comments to reach consensus and make the progress proceeds. But please realize there are always more pending issues than contributors are able to handle, and especially TiDB community is a global one, contributors reside all over the world and they might already be very busy with their own work and life. Please be patient! If your issue gets stale for some time, it's okay to ping the reviewers and asignees, or take it to [internal.tidb.io](https://internals.tidb.io) for more attention.
```

Comment:
```suggestion
Once your issue is created, other contributors might take part in. You can [/cc](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#cc) reviewers to request reviews. You need to discuss with them, provide more information they might want to know, address their comments to reach consensus and make the progress proceeds. But please realize there are always more pending issues than contributors are able to handle, and especially TiDB community is a global one, contributors reside all over the world and they might already be very busy with their own work and life. Please be patient! If your issue gets stale for some time, it's okay to ping the reviewers and assignees, or take it to [internal.tidb.io](https://internals.tidb.io) for more attention.
```

The concept "reviewer" is only active for a PR, not an issue. Do you think the original content a proper expression?

Reply:
FWIW I often use the sentence from https://infra.apache.org/patch.html

> Be patient if your patch is not applied as fast as you'd like (open source developers are all volunteers, often doing the development in their spare time) or a developer asks you to make changes to the patch. If you do not receive any feedback in a reasonable amount of time (say, a week or two), feel free to send a follow-up e-mail to the developer list.

Reply:
I can change the word to 'other participants'. I don't mean to give a sample sentence for others to  use. Just a warm remind here.